docs/functions.rst

```diff
@@ -874,19 +874,18 @@ Example:
       outputArtifacts:
         backupStats:
           keyValue:
-            mode: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputMode }}"
-            fileCount "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputFileCount }}"
-            size: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputSize }}"
+            mode: "{{ .Phases.BackupDataStatsFromObjectStore.Output.BackupDataStatsOutputMode }}"
```

Comment:
Where do we add mode to `Output`? Is this where it would be: https://github.com/kanisterio/kanister/blob/master/pkg/function/describe_backups.go#L94 ?

Reply:
This is in `BackupStats` https://github.com/kanisterio/kanister/blob/1f79078bba2f41f04680a736cb9acd6a1fa6cf17/pkg/function/backup_data_stats.go#L100

Reply:
Nevermind, this is the correct function: https://github.com/kanisterio/kanister/blob/master/pkg/function/backup_data_stats.go#L100.

Should the output keys be lowercase?

Reply:
I am seeing both cases in this file. Not sure which is better/correct?

Reply:
Lower case is better. Updating..